dashboard/db/schema.rb

```diff
@@ -346,13 +346,14 @@
     t.string   "email",              limit: 255, null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "school",             limit: 255
     t.string   "code",               limit: 255
+    t.string   "school",             limit: 255
```

Comment:
Does this kind of order change mess things up?  Not sure why my system does this from time to time.